app/controllers/events_controller.rb

```diff
@@ -0,0 +1,51 @@
+class EventsController < ApplicationController
+  before_action :set_event, only: [:destroy]
+
+  def index
+    @events = Event.all
```

Comment:
@kakada we can move Event.upcoming and Event.past to model instead. then we can hide all these logic

ej: def self.of_type(type) ...